### PR TITLE
As/build tensors

### PR DESCRIPTION
**Pull request by arsenetar**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -20,15 +20,14 @@ find_package(PkgConfig REQUIRED)
...
 set(PROTO_PATH "${PROJECT_SOURCE_DIR}/src/proto")
-set(PROTO_PATH2 "${PROJECT_SOURCE_DIR}/src/tensorizer/proto")
 set(GENERATED_PROTOBUF_PATH "${PROJECT_SOURCE_DIR}/gooseai")
 file(MAKE_DIRECTORY ${GENERATED_PROTOBUF_PATH})
 
 ## Python target support
 find_package(Python3 REQUIRED COMPONENTS Interpreter)
 
 set(PYBUILD_PATH "${PROJECT_BINARY_DIR}/pybuild")
-execute_process(COMMAND python3.9 -m venv ${PYBUILD_PATH}
```
Comment (wbrown) [inline CMakeLists.txt:31]: Using a Python later than 3.9 will break things.

File: .gitmodules [hunk 2]
```diff
@@ -1,3 +1,3 @@
-[submodule "src/tensorizer"]
-	path = src/tensorizer
-	url = https://github.com/coreweave/tensorizer.git
+[submodule "src/proto/tensorizer"]
+	path = src/proto/tensorizer
```
Comment (wbrown) [inline .gitmodules:2]: If we do this, we will need to adjust the library paths in things that use this.

**Review by wbrown — APPROVED** [review]
Minor comments about the changes.

File: .gitmodules (hunk 2 above)
Comment (arsenetar) [reply .gitmodules:2]: Do you know of something impacted? Golang based fetch from tensorizer directly, and does not appear to be an issue with the TS/JS partially due to it not working before.  Python would be about the only one to be concerned with this change from what I can tell depending on how the project was setup.

**Comment by cruhl** [comment]
This worked when I tried it, but I still need to try things on windows.

**Pull request merged**